recipe-server/client/control/components/HistoryItem.js

```diff
@@ -0,0 +1,71 @@
+import React, { PropTypes as pt } from 'react';
```

Comment:
This is extracting the HistoryItem component from `RecipeHistory` into its own file.